abdulfatai360/banka
# #165435822 Ch accounts endpoints migration

#### What does this PR do?
Migrate `account` endpoints to database

#### Description of Task to be completed?
- Have APIs for `accounts` endpoint working with database

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/43/head:ch-accounts-endpoints-migration-165435822`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-accounts-endpoints-migration-165435822`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
#165435822
